<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditSearchDotDialog</class>
 <widget class="QDialog" name="EditSearchDotDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>探索範囲１</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>探索範囲２</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>X方向</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Y方向</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxMoveDotX">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <number>99999999</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxMoveDotY">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>40</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <number>99999999</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxMoveDotXMM">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelUnit1">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>70</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>mm</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelUnit2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>70</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>mm</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxMoveDotYMM">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>70</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelUnit4">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>140</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>mm</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxMoveDotY2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>110</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <number>99999999</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelUnit3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>140</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>mm</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxMoveDotY2MM">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>140</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxMoveDotX2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <number>99999999</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxMoveDotX2MM">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOK">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>270</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>設定OK</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>270</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>キャンセル</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxJudgeWithBrDif">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>180</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>色でマッチング</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxCharacterMode">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>220</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>文字でマッチング</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
